en/core-libraries/text.rst

```diff
@@ -369,7 +369,7 @@ Output::
 Converting an Array to Sentence Form
 ====================================
 
-.. php:method:: toList(array $list, $and='and')
+.. php:method:: toList(array $list, $and='and', $separator = ', ')
```

Comment:
I wonder if the $and= also needs spacing around equals.
Or yours doesnt for consistency.